HCTextField/ViewController.swift

```diff
@@ -103,7 +99,7 @@ class ViewController: UIViewController, HCTextFieldDelegate {
 
     // MARK: - Private implementations
 
-    private func setRightView(for textField: HCTextField, type: HCTextFieldCheckType, errorMessage: String?) {
+    func setRightView(for textField: HCTextField, type: HCTextFieldCheckType, errorMessage: String?) {
```

Comment:
Let's put this method inside the textfield, so that the user doesn't have to implement this method.